ratingcurve/sklearn.py

```diff
@@ -119,8 +125,18 @@ def _nuts_inference(self, **kwargs):
         kwargs : dict
             Keyword arguments to pass to pm.sample
         """
+        import warnings
+
         nuts_kwargs = self.__default_nuts_kwargs.copy()
         nuts_kwargs.update(kwargs)
+
+        # Check to see if OS is Windows. If so, set cores=1 and print warning.
+        if sys.platform == 'win32' and nuts_kwargs['cores'] != 1:
+            nuts_kwargs.update({'cores': 1})
+            warnings.warn("PyMC NUTS sampler has known bug when fitting with "
+                          "more than one core on Windows. Overriding user "
+                          "input. Number of cores set to 1.", RuntimeWarning)
```

Comment:
Let's show this warning no matter what, but not override.
Perhaps link back to the github issue, if one exists.
```suggestion
        # Check to see if OS is Windows. If so, set cores=1 and print warning.
        if sys.platform == 'win32':
            nuts_kwargs.update({'cores': 1})
            warnings.warn("PyMC NUTS sampler has known bug when using "
                          "multiple cores on Windows. On Windows, the "
                          "number of cores is set to 1 by default.", RuntimeWarning)
```

Reply:
So, you are thinking to let it crash if the user tries to set `cores != 1`?

Reply:
I'm okay with that, so long as they see the warning. I'm okay with you overriding too, though it makes testing slightly harder.

The main thing is that users should see the warning whenever they fit nuts on Windows, so they know linux is better!

Reply:
Sounds good!